app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(elfs: list[Elf]) -> None:
+    for elf in elfs:
+        Elf.play_elf_song(elf)
```

Comment:
its better to call method from instance, not from class I guess

Reply:
agree